tracing/tracer.go

```diff
@@ -79,11 +79,12 @@ func (jl *jaegerLogger) Error(msg string) {
 func (jl *jaegerLogger) Infof(msg string, args ...interface{}) {
 	// TODO (madhu) Zap logger requires even number of arguments, add a key to each arg
 	// This is a hack in place till jaeger directly takes a zap logger
-	var logArgs []interface{}
+	targetLen := len(args) * 2
+	var targetArgs = make([]interface{}, targetLen)
```

Comment:
Alex, if capacity is not specified, it defaults to len. Just specifying length should do.